src/api/docker-stats.ts

```diff
@@ -52,6 +51,10 @@ export function getContainersStats(): Observable<any> {
       })
       .map(stats => observer.next({ type: 'containersStats', data: stats }))
       .subscribe();
+
+    return () => {
+      sub.unsubscribe();
```

Comment:
please check if `sub` exists before unsubscribing, just in case that we prevent errors.